Source/Wit/Public/Voice/Configuration/VoiceConfiguration.h

```diff
@@ -52,7 +52,7 @@ struct WIT_API FVoiceConfiguration
 	 * The minimum voice volume for keeping the voice input active
 	 */
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Keep Alive", meta=(ClampMin = 0, ClampMax = 1))
-	float KeepAliveMinimumVolume{0.01f};
+	float KeepAliveMinimumVolume{0.02f};
```

Comment:
will this improve something?

Reply:
yeah, I found that since I increased the mic sensitivity sometimes it was not dropping below the default 0.01 which mean that the mic input never auto-stopped